src/components/UserInfo/UserInfo.tsx

```diff
@@ -1 +1,16 @@
-export const UserInfo = () => {};
+import { User } from '../../types/type';
+
+type Props = {
+  user: User | null,
```

Comment:
```suggestion
  user: User,
```
better to render this component only when user exists